src/main/cpp/feedback/DragonLeds.cpp

```diff
@@ -20,6 +20,20 @@
 #include "feedback/DragonLeds.h"
 #include "utils/logging/debug/Logger.h"
```

Comment:
I was sort of hoping we would separate the feedback and the diagnotice leds like they did in the example